### PR TITLE
Minor update to getting started steps

### PR DESCRIPTION
**Pull request by bodrovphone**
After watching [Serverless for Frontend Developers](https://www.serverless.com/learn/courses/serverless-for-frontend-developers/) tutorial and then starting using this great component I've been a little confused and couldn't deploy as the official tutorial and this Readme.me file is out of sync.

Screenshot at Aug 01 14-43-15

An obvious thing for major could be not so obvious for others who might spend some precious time figuring it out.
There are a few more lines on this Readme.me file which references an outdated component like:

`component: serverless-next.js`

instead of being like:

component: "@sls-next/serverless-component@{version_here}" 

I'll be glad to change them all in scope of this PR - let me know.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,13 +65,20 @@ With a simplified architecture and no use of CloudFormation, there are no limits
 
 ### Getting started
 
+Install the component:
+
+```
+npm i @sls-next/serverless-component -D
```
Comment (danielcondemarin) [inline README.md:71]: You don't actually need to `npm install` the component. I've deliberately left this step out :)

File: README.md [hunk 2]
```diff
@@ -65,13 +65,20 @@ With a simplified architecture and no use of CloudFormation, there are no limits
...
+```
+
 Add your next application to the serverless.yml:
 
 ```yml
 # serverless.yml
 
 myNextApplication:
   component: "@sls-next/serverless-component@{version_here}" # it is recommended you pin the latest stable version of serverless-next.js
+  # replace {version_here} with the one from package.json
```
Comment (danielcondemarin) [inline README.md:81]:
```suggestion
  # it is recommended you pin the latest stable version of serverless-next.js
  # replace {version_here} with a stable npm version available
  component: "@sls-next/serverless-component@{version_here}" 
```

File: README.md (hunk 1 above)
Comment (bodrovphone) [reply README.md:71]: Thanks @danielcondemarin.
Can you tell me why I've got the error (see description screenshot) when running serverless deploy?

Comment (danielcondemarin) [reply README.md:71]: That version is genuinely not in the registry:

image

Comment (bodrovphone) [reply README.md:71]: oops :)
Have no idea where did I take that version from. Probably my Friday imagination :)

Is it true that `component: serverless-next.js` should be replaced with `component: "@sls-next/serverless-component@{version_here}"` across Readme file ?

Comment (danielcondemarin) [reply README.md:71]: Yes that's true. serverless-next.js used to be the old name of the package before I created the org @sls-next :)

**Pull request closed without merging**